api/tests/routes/backoffice/offerers_test.py

```diff
@@ -1618,12 +1618,39 @@ def test_get_offerer_addresses(self, authenticated_client, offerer):
         rows = html_parser.extract_table_rows(response.data)
         assert len(rows) == 2
 
-        assert rows[0]["Intitulé"] == "Deuxième adresse"
-        assert rows[0]["Adresse"] == "5 Bd Poissonnière 75002 Paris"
+        assert rows[0]["Intitulé"] == "Première adresse"
+        assert rows[0]["Adresse"] == "3 Bd Poissonnière 75002 Paris"
         assert rows[0]["Localisation"] == "48.87055, 2.34765"
 
-        assert rows[1]["Intitulé"] == "Première adresse"
-        assert rows[1]["Adresse"] == "3 Bd Poissonnière 75002 Paris"
+        assert rows[1]["Intitulé"] == "Deuxième adresse"
+        assert rows[1]["Adresse"] == "5 Bd Poissonnière 75002 Paris"
+        assert rows[1]["Localisation"] == "48.87055, 2.34765"
+
+    def test_offerer_addresses_linked_to_venues_should_display_common_name(self, authenticated_client, offerer):
+        oa_linked_to_venue = offerers_factories.OffererAddressFactory(
+            offerer=offerer, address__street="3 Bd Poissonnière", label=None
+        )
+        venue = offerers_factories.VenueFactory(managingOfferer=offerer, offererAddress=oa_linked_to_venue)
+        offerers_factories.OffererAddressFactory(
+            offerer=offerer, label="Autre localisation", address__street="5 Bd Poissonnière"
+        )
```

Comment:
Je proposerai bien une 2me OA à la même adresse pour tester la concaténation des labels:
```suggestion
        offerers_factories.OffererAddressFactory(
            offerer=offerer, label="Autre localisation", address__street="5 Bd Poissonnière"
        )
        offerers_factories.OffererAddressFactory(
            offerer=offerer, label="3me localisation", address__street="5 Bd Poissonnière"
        )
```

Reply:
C'est fait !